gef.py

```diff
@@ -7359,10 +7399,18 @@ def do_invoke(self, argv, *args, **kwargs):
         if args.big_endian:
             endian_s = "big"
 
-        if arch_s.upper() not in self.valid_archs or mode_s.upper() not in self.valid_modes:
-            raise AttributeError("invalid arch/mode")
+        if arch_s.upper() not in self.valid_arch_modes:
```

Comment:
let's just set mode and arch to upper earlier in the func so we don't have to keep calling the method.

Reply:
I thought about that but then also the output gef shows will be upper (unless we create a new var just for that). So should I still call upper on it ealier? What do you think?